### PR TITLE
Add Protected Audience reporting doc

### PR DESCRIPTION
**Pull request by kevinkiklee**
This PR adds the Protected Audience reporting doc

Preview: https://pr-7389-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/protected-audience-api/reporting/

### CODE REVIEW

File: site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+
+{% Aside %}
+
+We recommend you read the documentation on [Private Aggregation](/docs/privacy-sandbox/private-aggregation/), [Fenced Frames](/docs/privacy-sandbox/fenced-frame/), [Shared Storage](/docs/privacy-sandbox/shared-storage/), and [Attribution Reporting](/docs/privacy-sandbox/attribution-reporting/) before continuing, as core concepts are expanded on in this document.
+
+{% endAside %}
+
+## Protected Audience API reporting overview
+
+{% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/R35UFscm9XdMmtco0BYH.png", alt="Protected Audience workflow", width="800", height="414" %}
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md:30]: This should probably also have a figcaption.

File: site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+```js
+// Ad frame
+window.fence.reportEvent({
+  'eventType': 'click',
+  'eventData': JSON.stringify({'clickX': '123', 'clickY': '456'}),
+  'destination':['buyer', 'seller']
+});
+```
+
+The Fenced Frames Ads Reporting API will also be available until at least 2026 for [the same reasons as win reporting](/docs/privacy-sandbox/fledge-api/feature-status/#event-level-auction-win-reporting).
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md:319]: This link is broken in the preview.

File: site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+
+### Attribution report
+
+An _attribution report_ allows you to associate a conversion on a website with an ad that was chosen from a Protected Audience API auction. For example, a user may click on a product ad you serve, redirected to the advertiser’s site, make a purchase there, and you are interested in attributing the purchase to the ad that was shown. The [Attribution Reporting API](https://github.com/WICG/attribution-reporting-api/tree/main) will be integrated with Protected Audience API to combine the auction data from the publisher site and the conversion data from the advertiser site. 
+
+While we design a more permanent solution, you can use the [Ads Reporting API](#bookmark=kix.bf7i1c40kape) for fenced frames as a transitional mechanism for generating an event-level and aggregatable report with Attribution Reporting. Note that these reports are for measuring conversion, and are separate from the event-level and aggregatable engagement reports generated from the auction and the ad frame. We will publish an explainer for a more permanent solution when ready.
```
[the remark this answers is not in the document]
Comment (kevinkiklee) [reply site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md]: arg, it's what the gdoc -> markdown plugin adds. i thought i replaced all those. 

fixed now.

File: site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md (hunk 2 above)
Comment (kevinkiklee) [reply site/en/docs/privacy-sandbox/protected-audience-api/reporting/index.md]: Updated

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**